src/EnergyPlus/UnitarySystem.cc

```diff
@@ -16066,7 +16066,8 @@ namespace UnitarySystems {
         }
 
         if (this->m_SuppCoilExists) {
-            if (this->m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingElectric) {
+            if (this->m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingElectric ||
+                this->m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingElectric_MultiStage) {
                 suppHeatingPower = state.dataHVACGlobal->SuppHeatingCoilPower;
             }
         }
```

Comment:
Multistage supplemental electric heating coil electricity rate was not included in the Unitary System parent total electricity rate. This change fixes that.

Reply:
Looks good.